src/NLog/NewRelic.LogEnrichers.NLog/NewRelicJsonLayout.cs

```diff
@@ -79,6 +79,7 @@ protected override void InitializeLayout()
             IncludeMdlc = false;
             RenderEmptyObject = false;
             SuppressSpaces = true;
+            MaxRecursionLimit = 1;
```

Comment:
We should also do this in the .ctor of this class where we create another JsonLayout instance.

Reply:
🎉